common/src/main/java/org/apache/rocketmq/common/UtilAll.java

```diff
@@ -765,4 +765,31 @@ private static void createDirIfNotExist(String dirName) {
             STORE_LOG.info(dirName + " mkdir " + (result ? "OK" : "Failed"));
         }
     }
+
+    public static long calculateFileSizeInPath(File path) {
+        long size = 0;
+        try {
+            if (!path.exists()) {
+                return 0;
+            }
+            if (path.isFile()) {
+                return path.length();
+            }
+            if (path.isDirectory()) {
+                File[] files = path.listFiles();
+                if (files != null) {
+                    for (File file : files) {
+                        long fileSize = 0;
+                        fileSize = calculateFileSizeInPath(file);
```

Comment:
It's unnecessary to initialize `fileSize` as 0 as it's assigned right after initialization.